contents/handbook/company/management.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Management at PostHog
+sidebar: Handbook
+showTitle: true
+---
+
+As we grow, we'll increase the number of managers at PostHog. Here's what a manager at PostHog looks like.
+
+## Defining the role of manager
+
+A manager at PostHog has these tasks
```

Comment:
Q: What about providing feedback?

Reply:
Another good q - do we do feedback like we did before? 360 degrees with everyone on the team there? Or is the manager responsible for feedback in private?

I actually really liked the open feedback that we did before.

Reply:
I would personally have a strong preference for the manager to not be responsible for feedback, in that it can reinforce hierarchy and make people less inclined to tread on toes.

Reply:
+1 to the open feedback. I'm imagining that will take shape in smaller groups that are working together as the company gets bigger so that could be the best formal feedback setup. Then normally, the manager will have 1 on 1s which will contain some amount of unstructured feedback

Reply:
Love it, will add it to the list of things managers don't do (though they will in a personal capacity if that makes sense)